pages/index.tsx

```diff
@@ -12,41 +14,10 @@ const Home: NextPage = () => (
       <meta name="description" content="Generated by create next app" />
       <link rel="icon" href="/upper_room_media_icon.png" />
     </Head>
-
-    <main className={styles.main}>
-      <h1 className={styles.title}>
-        Welcome to <a href="https://nextjs.org">Next.js!</a>
-      </h1>
-
-      <p className={styles.description}>
-        Get started by editing <code className={styles.code}>pages/index.js</code>
-      </p>
-
-      <div className={styles.grid}>
-        <a href="https://nextjs.org/docs" className={styles.card}>
-          <h2>Documentation &rarr;</h2>
-          <p>Find in-depth information about Next.js features and API.</p>
-        </a>
-
-        <a href="https://nextjs.org/learn" className={styles.card}>
-          <h2>Learn &rarr;</h2>
-          <p>Learn about Next.js in an interactive course with quizzes!</p>
-        </a>
-
-        <a href="https://github.com/vercel/next.js/tree/canary/examples" className={styles.card}>
-          <h2>Examples &rarr;</h2>
-          <p>Discover and deploy boilerplate example Next.js projects.</p>
-        </a>
-
-        <a
-          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          className={styles.card}
-        >
-          <h2>Deploy &rarr;</h2>
-          <p>Instantly deploy your Next.js site to a public URL with Vercel.</p>
-        </a>
-      </div>
-    </main>
+    <div className={styles.rotating}>
+      <Image src={Gear} height={300} width={300} />
+    </div>
+    <h1 className={styles.loading}>Work in progress</h1>
```

Comment:
Can you please left align the text so that it does not shift when the ellipses pop in as shown in this video

https://user-images.githubusercontent.com/21961090/200678028-d3ec1cd7-2f8e-4a29-a3a0-39cc642069ed.mov